tox.ini

```diff
@@ -126,6 +126,9 @@ deps =
     cassandra38: cassandra-driver>=3.8,<3.9
     celery31: celery>=3.1,<3.2
     celery40: celery>=4.0,<4.1
+    celery41: celery>=4.1,<4.2
+    # TODO[manu] update to a stable version of Celery
+    celery42: celery==4.2.0rc2
```

Comment:
Can you update it with the latest RC?
```
celery42: celery==4.2.0rc3
```

Reference: https://github.com/celery/celery/releases/tag/v4.2.0rc3